src/sentry/static/sentry/app/components/charts/lineChart.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import theme from 'app/utils/theme';
+
+import BaseChart from './baseChart';
+import Tooltip from './components/tooltip';
+import XAxis from './components/xAxis';
+import YAxis from './components/yAxis';
+import LineSeries from './series/lineSeries';
+
+export default class lineChart extends React.Component {
+  static propTypes = {
+    ...BaseChart.propTypes,
+  };
+
+  defineSeries = ([key, value], idx) => {
+    return LineSeries({
+      name: key,
+      type: 'line',
```

Comment:
We don't need to set `type: line` since that will be always overwritten by LineSeries to be `line`

Reply:
thank you!